integ/components/deadline/common/scripts/bash/deploy-utils.sh

```diff
@@ -44,7 +52,11 @@ function destroy_component_stacks () {
   run_aws_interaction_hook
 
   echo "Destroying test app $COMPONENT_NAME..."
-  npx cdk destroy "*" -f
+  if [ "${RUN_TESTS_IN_PARALLEL}" = true ]; then
+    npx cdk destroy "*" -f > "$INTEG_TEMP_DIR/${COMPONENT_NAME}_destroy.txt" 2>&1
```

Comment:
I don't see these `*_destroy.txt` log file being used anywhere. Can we output them if something goes wrong during the destroy?

Reply:
I thought destroying log will not be very useful, but on other hand there is no useless information. Added.